backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/CodeGenerator.kt

```diff
@@ -125,6 +124,10 @@ internal inline fun<R> generateFunction(codegen: CodeGenerator,
             startLocation,
             endLocation,
             function), code)
+
+    // To perform per-function validation.
+    if (false)
```

Comment:
We definitely need a handle (CLI flag?) for that. But it can be added in a separate PR.